osf-cas/templates/configmap.yaml

```diff
@@ -805,7 +812,7 @@ services/preprints-osf.json: |-
 {{- $domainPrefix := .Values.preprintProviderDomains.prefix -}}
 {{- $domainSuffix := .Values.preprintProviderDomains.suffix -}}
 {{- range $key, $val := (include "cas.jetty.preprint-services" . | fromYaml) }}
-services/preprints-{{ $key }}.json: |-
+services/{{ $key }}-{{ $val.id }}.json: |-
```

Comment:
@mattclark The only change that may break the build since I can't test building the files from templates. Does this look correct. For example, the current file name for a branded preprints is `preprints-socarxiv.json` and after this update it will be (or is expected to be `socarxiv-203948234207241.json`.